### PR TITLE
Version 0.1.0 for initial review

### PR DESCRIPTION
**Pull request by chinmaygarde**

### CODE REVIEW

File: lib/newton.dart [hunk 1]
```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+library newton;
+
+import 'dart:math' as Math;
```
Comment (abarth) [inline lib/newton.dart]: s/Math/math/

File: lib/src/scroll_simulation.dart [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+part of newton;
+
+/// Simulates kinetic scrolling behavior between a leading and trailing
+/// boundary. Friction is applied within the extends and a spring action applied
+/// at the boundaries. This simulation can only step forward.
+class ScrollSimulation extends SimulationGroup {
+  final double _leadingExtent;
+  final double _trailingExtent;
+  final SpringDesc _springDesc;
```
Comment (abarth) [inline lib/src/scroll_simulation.dart]: Desc -> Description?  Generally, we've been preferring complete words.

File: lib/src/spring_simulation.dart [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+  /// The spring constant (k)
+  final double springConstant;
+
+  /// The damping coefficient.
+  /// Note: Not to be confused with the damping ratio. Use the separate
+  ///       constructor provided for this purpose
+  final double damping;
+
+  SpringDesc(this.mass, this.springConstant, this.damping);
```
Comment (abarth) [inline lib/src/spring_simulation.dart]: I would probably use named arguments here given that these are all doubles and will be hard to tell apart at call sites.

File: lib/src/spring_solution.dart [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+    }
+
+    return null;
+  }
+
+  SpringType get type;
+}
+
+class _CriticalSolution implements _SpringSolution {
+  final double _r, _c1, _c2;
```
Comment (abarth) [inline lib/src/spring_solution.dart]: These variables are pretty mysterious.  Can we give them better names or add a comment explaining them?

File: lib/newton.dart (hunk 1 above)
Comment (chinmaygarde) [inline lib/newton.dart]: ack

File: lib/src/scroll_simulation.dart (hunk 2 above)
Comment (chinmaygarde) [inline lib/src/scroll_simulation.dart]: ack

File: lib/src/spring_simulation.dart (hunk 3 above)
Comment (chinmaygarde) [inline lib/src/spring_simulation.dart]: ack

**Pull request closed without merging**